src/bloqade/emulate/ir/state_vector.py

```diff
@@ -34,7 +34,7 @@ class RabiOperator:
     phase: Optional[Callable[[float], float]] = None
 
     def dot(self, register: NDArray, time: float):
-        amplitude = self.amplitude(time)
+        amplitude = self.amplitude(time) / 2
```

Comment:
can you explain why this is div by 2?

Reply:
that's how we define the Ryberg Hamiltonian

Reply:
Remember this guy? https://github.com/QuEraComputing/Bloqade.jl/blob/c219ebfbfc750e260e52c9b57984582b4e71dfcf/lib/BloqadeExpr/src/types.jl#L689C8-L689C16